static/src/stylesheets/module/experiments/_giraffe.scss

```diff
@@ -1,20 +1,32 @@
 .giraffe {
     @include f-header;
     width: 100%;
-    display: block;
-    margin-bottom: 1em;
-    border-top: 1px dotted $neutral-4;
+    padding: 5px 5px 12px;
```

Comment:
`padding: $gs-baseline / 2 $gs-gutter / 4 $gs-baseline`? I believe as much as possible we should tie layout to the grid vars, i.e. `$gs-baseline` for vertical rythm and `$gs-gutter` and `gs-span` for horizontal rythm.